ViAn/Filehandler/filehandler.h

```diff
@@ -36,10 +27,11 @@ class FileHandler
 {
 public:
     FileHandler();
-    std::string workSpace;
     //
     //  Project manipulation
     //
+    std::string workSpace;
```

Comment:
This is not following the snake_case standard we have. Look through all new code for this please.